draft-yasskin-http-origin-signed-responses.md

```diff
@@ -516,37 +516,38 @@ reading):
 
 ~~~http
 Signature: sig1;
+  validityUrl="https://example.com/resource.validity.1511128380";
   sig=*MEUCIQDXlI2gN3RNBlgFiuRNFpZXcDIaUpX6HIEwcZEc0cZYLAIga9DsVOMM+g5YpwEBdGW3sS+bvnmAJJiSMwhuBdqp5UY;
   integrity="mi";
-  validityUrl="https://example.com/resource.validity.1511128380";
   certUrl="https://example.com/certs";
   certSha256=*W7uB969dFW3Mb5ZefPS9Tq5ZbH5iSmOILpjv2qEArmI;
   date=1511128380; expires=1511560380,
-  sig2;
+ sig2;
+  validityUrl="https://example.com/resource.validity.1511128380";
   sig=*MEQCIGjZRqTRf9iKNkGFyzRMTFgwf/BrY2ZNIP/dykhUV0aYAiBTXg+8wujoT4n/W+cNgb7pGqQvIUGYZ8u8HZJ5YH26Qg;
   integrity="mi";
-  validityUrl="https://example.com/resource.validity.1511128380";
   certUrl="https://example.com/certs";
-  certSha256=*kQAA8u33cZRTy7RHMO4+dv57baZL48SYA2PqmYvPPbg;
+  certSha256=*W7uB969dFW3Mb5ZefPS9Tq5ZbH5iSmOILpjv2qEArmI;
   date=1511301183; expires=1511905983,
-  sig3;
+ sig3;
+  validityUrl="https://THIRDPARTY.com/resource.validity.1511128380";
```

Comment:
Ahhh, I don't think we'd want to use anything other than the IANA-reserved domains, especially in a draft, so I think you were doing great with thirdparty.example.com, which is why I mentioned the suffix part :)

Put more precise: I do not think we should deviate from https://tools.ietf.org/html/rfc6761#section-6.5

Reply:
Thanks for expressing that opinion. Fixed.